main.js

```diff
@@ -0,0 +1,188 @@
+/*Heather Sleyster
+sleyster_a02b
+INFO 2134
+Thoendel
+03/17/2020
+*/
+window.addEventListener('load', function() {
+/*These statements assign the Form element to the form variable 
+and the select element named 'meal' to the select variable.
+*/
+const form = document.getElementById('contactForm');
+const meal = document.getElementByName('meal');
+
+/*The first statement assigns the elements with the name of mealOptionsHolder
+to the variable mealOption.This if statement checks the value of the 
+selected meal and if it is empty removes the 'visible' class from the form. 
+If breakfast is selected the visible class will be added to that option only, 
+if lunch is selected the visible class will be added to that option only,
+if dinner is selected the visible class will be added to that option.
+*/
+const mealOption = document.getElementsByName('mealOptionsHolder');
+    if (meal.value == 'breakfast'){
+        const breakfastOptions = getElementById('breakfastOptions');
+        breakfastOptions.classList.remove('hidden');
+        breakfastOptions.classList.add('visible');
+    }
+        else if (meal.value == 'lunch'){
+            const lunchOptions = getElementById('lunchOptions');
+            lunchOptions.classList.remove('hidden');
+            lunchOptions.classList.add('visible');
+        }
+            else if (meal.value == 'dinner') {
+                const dinnerOptions = getElementById('dinnerOptions');
+                dinnerOptions.classList.remove('hidden');
+                dinnerOptions.classList.add('visible');
+            }
```

Comment:
we want this to happen when there is a change to meal. We need a listener to make that happen. The code is solid - but not being applied because we aren't listening for that change.